Wikipedia/Code/MWKSection.m

```diff
@@ -280,7 +280,7 @@ - (NSString*)summary {
 - (nullable NSArray<MWKTitle*>*)disambiguationTitles {
     NSArray* textNodes = [self elementsInTextMatchingXPath:WMFSectionDisambiguationTitlesXPathSelector];
     return [textNodes wmf_mapAndRejectNil:^id (TFHppleElement* node) {
-        if (node.text.length == 0 || [node.text containsString:@"redlink=1"]) {
+        if (node.text.length == 0 || [node.text containsString:@"redlink=1"] || [node.text containsString:@"cite_note"]) {
```

Comment:
also: `#cite_note` is just one instance of a string w/o the "internal link prefix" (as the assertion says `MWKTitle` is expecting).  maybe it would be safer to just filter out all nodes w/o the internal link prefix (i.e. `/wiki/...`)?